### ISSUE TITLE
Changing compileOptions to 11 cause  Package not found

### ISSUE THREAD

**Issue by yatw** [googlecodelabs/android-paging#208]
Hello, I notice if change build.gradle to java 11, all the View package cannot be found.

<img width="1521" alt="Screen Shot 2021-12-07 at 15 57 18" src="https://user-images.githubusercontent.com/19565177/145124228-ff0cf322-6faa-4566-9527-1d3d159972e3.png">

This is fresh new clone from branch step13-19_network_and_database with the compile option change. I know not just this repo will run into this problem But I would like to take this chance to ask what is causing this error.

Basically all the view and widget package cannot be found. And all the databinding generated class cannot find symbols.
